instrumentation/cassandra/cassandra-3.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/cassandra/v3_0/TracingSession.java

```diff
@@ -53,7 +52,7 @@ public Session apply(Session session) {
             return new TracingSession(session);
           }
         },
-        directExecutor());
```

Comment:
`directExecutor` was only introduced in guava 18.0, and cassandra 3.0.0 depends on guava 16.0, so muzzle (rightly) not happy with usage of it